engine/src/main/java/org/destinationsol/game/screens/MainScreen.java

```diff
@@ -503,6 +511,22 @@ public boolean isAbility() {
         return shipControl.isAbility();
     }
 
+    public boolean isCameraUp(){
```

Comment:
Similar to a set of comments above: should be renamed to `isCameraUpControlEnabled` or similar.